### PR TITLE
Added a line

### PR DESCRIPTION
**Pull request by ArinGhazarian**
First pr.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Test
 A test repository just to play around with various GitHub features.
+
+Added a line.
```
Comment (ArinGhazarian) [inline README.md:4]: Fix this please

Comment (brianaj) [reply README.md:4]: respond to this

**Review by brianaj — CHANGES_REQUESTED** [review]
This is a different comment type?

**Comment by brianaj** [comment]
Let's see if this is resolvable from the "Conversation" thread

**Comment by brianaj** [comment]
> Let's see if this is resolvable from the "Conversation" thread

File: README.md (hunk 1 above)
Comment (ArinGhazarian) [reply README.md:4]: done

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # Test
 A test repository just to play around with various GitHub features.
+
```
Comment (teja-nanduri) [inline README.md:3]: Test

File: README.md (hunk 1 above)
Comment (teja-nanduri) [inline README.md:4]: another text

Comment (ArinGhazarian) [reply README.md:4]: Oh really!

Comment (ArinGhazarian) [inline README.md:4]: Oops forgot about this one!

**Review by ArinGhazarian — COMMENTED** [review]
Testing

File: README.md (hunk 1 above)
Comment (ArinGhazarian) [reply README.md:4]: What about this?

**Pull request closed without merging**